Mobile/Android/Templates/1209-material_buttons.md

```diff
@@ -0,0 +1,33 @@
+### Uploads view
+
+#### Pr: https://github.com/owncloud/android/pull/1209
+
+Devices:
```

Comment:
It's convenient to test in an Android 4.x devices, and in other device that could be 5.x or 6.x